packages/shared/src/api/alephiumBaseQuery.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseQueryFn } from '@reduxjs/toolkit/query'
+
+export const alephiumBaseQuery: BaseQueryFn<() => Promise<any>, unknown, { message: string }> = async (fn) => {
+  try {
+    const result = await fn()
+    return { data: result }
+  } catch (error: any) {
+    return { error: { message: error.message || 'An error occurred' } }
```

Comment:
The `error` returned from web3 does not have a `message` property: https://github.com/alephium/alephium-web3/blob/master/packages/web3/src/api/utils.ts#L25